### PR TITLE
V1.0.0 docs prep

### PR DESCRIPTION
**Pull request by MichaelClifford**
Synced chatbot, codegen, and summarizer README's in preparation for v1 release. 

They should now only differ in their references to their respective directory or app name.

### CODE REVIEW

File: recipes/natural_language_processing/codegen/README.md [hunk 1]
```diff
@@ -22,111 +31,131 @@ In order to build this application we will need a model, a Model Service and an
...
+There are a number of options for quantization level, but we recommend `Q4_K_M`. 
 
 The recommended model can be downloaded using the code snippet below:
 
 ```bash
-cd models
-wget https://huggingface.co/TheBloke/Mistral-7B-Code-16K-qlora-GGUF/resolve/main/mistral-7b-code-16k-qlora.Q4_K_M.gguf
-cd ../
+cd ../../../models
+curl -sLO https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf
```
Comment (Gregory-Pereira) [inline recipes/natural_language_processing/codegen/README.md]: Is this intentional changing the model name?

Comment (MichaelClifford) [reply recipes/natural_language_processing/codegen/README.md]: nope, good catch.

File: recipes/natural_language_processing/codegen/README.md [hunk 2]
```diff
@@ -22,111 +31,131 @@ In order to build this application we will need a model, a Model Service and an
...
-podman build -t llamacppserver -f base/Containerfile .
+# from path model_servers/llamacpp_python from repo containers/ai-lab-recipes
+make build
 ```
+Checkout the [Makefile](../../../model_servers/llamacpp_python/Makefile) to get more details on different options for how to build.
 
+## Deploy the Model Service
 
-### Deploy the Model Service
+The local Model Service relies on a volume mount to the localhost to access the model files. It also employs environment variables to dictate the model used and where its served. You can start your local Model Service using the following `make` command from `model_servers/llamacpp_python` set with reasonable defaults:
```
Comment (Gregory-Pereira) [inline recipes/natural_language_processing/codegen/README.md:99]: Not sure about the wording of this first sentence. Localhost typically comes with the connotation of the browser / internet, where as the volume mount you are discussing is at a container level. Also lets add a link to the path: `(../../../model_servers/llamacpp_python/)`

**Review by Gregory-Pereira — APPROVED** [review]
/lgtm

**Comment by Gregory-Pereira** [comment]
Documentation updates related to #262

**Pull request merged**